content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md

```diff
@@ -168,3 +168,11 @@ To delete them, follow Docker's instructions to [Uninstall Docker Engine](https:
 {{< caution >}}
 Docker's instructions for uninstalling Docker Engine create a risk of deleting containerd. Be careful when executing commands.
 {{< /caution >}}
+
+## Uncordon the node 
+
+```shell
+kubectl uncordon <node-to-uncordon>
+```
+
+Replace `<node-to-uncordon>` with the name of your node you previously drained.
```

Comment:
nit: Replace `<node-to-uncordon>` with the name of the node you previously drained.